examples/imagenet/train_imagenet.py

```diff
@@ -72,20 +71,9 @@ def get_example(self, i):
 
 
 def parse_device(args):
```

Comment:
I think this function should be removed.
You can simply write `device = chainer.get_device(args.gpu or args.device)`.

Reply:
- `bool(0)` evaluates to `False` :cry:
- `args.device or args.gpu` looks OK, if it's fine to prioritize the `--device` arg.

Reply:
I see... I don't think that works.
The default value of `--device` would have to be `''` (an empty string) which evaluates to `False`. That default value would not work by itself. As `--gpu` is deprecated, we should not rely on it as a fallback.

Another possibility is to write `device = chainer.get_device(args.device if args.gpu is None else args.gpu)`.

Reply:
I'll try `dest='device'`.